mPower2/mPower2/study-burst-task.json

```diff
@@ -0,0 +1,22 @@
+{
+    "identifier": "study-burst-task",
+    "type": "studyBurst",
+    "numberOfDays": 14,
+    "activeTaskGroupIdentifier": "Measuring",
+    "competionTasks" : [
```

Comment:
comp**l**etionTasks

Reply:
This file is WIP. Added so that the project build won't fail, but should be ignored for now.